### ISSUE TITLE
Library not loaded error message in Snow Leopard OSX 10.8.6

### ISSUE THREAD

**Issue by wuworkshop** [fourseven/meteor-scss#56]
Very new to Meteor so pardon my newbie issue. I just created a new meteor test app and added only fourseven:scss and I get this error message in Terminal when I try to start up the app:

```
While loading plugin `fourseven:scss` from package `fourseven:scss`:
   module.js:356:32:
   dlopen(/Users/alice/.meteor/packages/fourseven_scss/.1.0.0.umakgb++os.osx.x86_64+web.browser+web.cordova/plugin.fourseven:scss.os.osx.x86_64/npm/fourseven:scss/node_modules/node-sass/bin/darwin-x64-v8-3.14/binding.node,
   1): Library not loaded: /usr/lib/libc++.1.dylib
   Referenced from:
   /Users/alice/.meteor/packages/fourseven_scss/.1.0.0.umakgb++os.osx.x86_64+web.browser+web.cordova/plugin.fourseven:scss.os.osx.x86_64/npm/fourseven:scss/node_modules/node-sass/bin/darwin-x64-v8-3.14/binding.node
   Reason: image not found
   at Module.load (module.js:356:32)
   at Function.Module._load (module.js:312:12)
   at Module.require (module.js:364:17)
   at require (module.js:380:17)
   at requireBinding
   (/Users/alice/.meteor/packages/fourseven_scss/.1.0.0.umakgb++os.osx.x86_64+web.browser+web.cordova/plugin.fourseven:scss.os.osx.x86_64/npm/fourseven:scss/node_modules/node-sass/sass.js:18:14)
   at Object.<anonymous>
   (/Users/alice/.meteor/packages/fourseven_scss/.1.0.0.umakgb++os.osx.x86_64+web.browser+web.cordova/plugin.fourseven:scss.os.osx.x86_64/npm/fourseven:scss/node_modules/node-sass/sass.js:25:15)
   at Module._compile (module.js:456:26)
   at Object.Module._extensions..js (module.js:474:10)
   at Module.load (module.js:356:32)
   at Function.Module._load (module.js:312:12)
   at Module.require (module.js:364:17)
   at require (module.js:380:17)
   at Object._.extend.Npm.require
   (/Users/alice/.meteor/packages/meteor-tool/.1.0.38.1sx6a09++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/bundler.js:1327:22)
   at Package (packages/fourseven:scss/plugin/compile-scss.js:2:1)
   at <runJavaScript-51>:102:4
   at <runJavaScript-51>:109:3
```
Any ideas? Should this package work on OSX 10.6.8? As a test, I tried a different scss package (bbfc:scss) and that worked just fine for me. Any help would be greatly appreciated.

**Comment by fourseven** [comment]
Sorry, this release was built on a Mac OS 10.10 machine, so the libraries are different. Ironically, `bbfc:scss` is actually my code, just re-released by another meteor user, but was an earlier build. I'll release the next version using the meteor build chain, but that might still be a problem as I'm not sure what version of Mac OS they supply. 

I'd say either continue using the bbfc version, or upgrade your machine - as I'm guessing you're not running a snow leopard server, so it's only a problem for your development environment.

**Comment by fourseven** [comment]
@wuworkshop I've just released 2.0.0-beta_1 through the official means, can you try that release please?

**Comment by wuworkshop** [comment]
Assuming I did this correctly, I typed ```meteor add fourseven:scss@="2.0.0-beta_1"```

I get the same error message as before. 

```
While loading plugin `fourseven:scss` from package `fourseven:scss`:
   module.js:356:32:
   dlopen(/Users/alice/.meteor/packages/fourseven_scss/.2.0.0-beta_1.lghr56++os.osx.x86_64+web.browser+web.cordova/plugin.fourseven:scss.os.osx.x86_64/npm/fourseven:scss/node_modules/node-sass/vendor/darwin-x64/binding.node,
   1): Library not loaded: /usr/lib/libc++.1.dylib
   Referenced from:
   /Users/alice/.meteor/packages/fourseven_scss/.2.0.0-beta_1.lghr56++os.osx.x86_64+web.browser+web.cordova/plugin.fourseven:scss.os.osx.x86_64/npm/fourseven:scss/node_modules/node-sass/vendor/darwin-x64/binding.node
   Reason: image not found
   at Module.load (module.js:356:32)
   at Function.Module._load (module.js:312:12)
   at Module.require (module.js:364:17)
   at require (module.js:380:17)
   at Object.<anonymous>
   (/Users/alice/.meteor/packages/fourseven_scss/.2.0.0-beta_1.lghr56++os.osx.x86_64+web.browser+web.cordova/plugin.fourseven:scss.os.osx.x86_64/npm/fourseven:scss/node_modules/node-sass/lib/index.js:211:15)
   at Module._compile (module.js:456:26)
   at Object.Module._extensions..js (module.js:474:10)
   at Module.load (module.js:356:32)
   at Function.Module._load (module.js:312:12)
   at Module.require (module.js:364:17)
   at require (module.js:380:17)
   at Object._.extend.Npm.require
   (/Users/alice/.meteor/packages/meteor-tool/.1.0.38.1sx6a09++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/tools/bundler.js:1327:22)
   at Package (packages/fourseven:scss/plugin/compile-scss.js:2:1)
   at <runJavaScript-51>:119:4
   at <runJavaScript-51>:126:3
```
I appreciate the attempt to fix this issue. I think at this point my options are to get a new laptop or go the Nitrous.io route.
